packages/components/src/stories/development/migration/09-migration-from-9x-to-10x.stories.mdx

```diff
@@ -14,6 +14,10 @@ import { Meta } from '@storybook/addon-docs'
 The following workflow walks you through the migration steps.
 Note that the actual steps required for your project may vary, and these steps should be treated as general guidance rather than strict instructions.
 
+## Radio
+
+The **checked** property of the `bal-radio` has been removed. Please use the `bal-radio-group` an its property **value**.
```

Comment:
I think this is the wrong file. The property was available in version 10 and was removed between version 10 and 12.

Reply:
Hi @chester23491 

Thanks adjusted that and add an example with before and after